broadlink/climate.py

```diff
@@ -238,3 +240,307 @@ def set_schedule(self, weekday: List[dict], weekend: List[dict]) -> None:
             input_payload.append(int(weekend[i]["temp"] * 2))
 
         self.send_request(input_payload)
+
+
+class sq1(device):
+    """Controls Tornado SMART X SQ series air conditioners."""
+
+    REQUEST_PREFIX = bytes([0xbb, 0x00, 0x06, 0x80])
+    RESPONSE_PREFIX = bytes([0xbb, 0x00, 0x07, 0x00])
```

Comment:
Move REQUEST_PREFIX to self._encode and RESPONSE_PREFIX to self._decode. I think we can assume:
```python3
    REQUEST_PREFIX = bytes([0xbb, 0x00, 0x06, 0x80, 0x00, 0x00])
    RESPONSE_PREFIX = bytes([0xbb, 0x00, 0x07, 0x00, 0x00, 0x00])
```